### PR TITLE
passphrase-input

### PR DESCRIPTION
**Pull request by danimoh**
I merged x-input, x-passphrase-input, x-passphrase-setter, x-passphrase-getter and x-passphrase-indicator from old keyguard together into one component.
One could consider splitting up a passphrase-chooser component again.

### CODE REVIEW

File: demos/PassphraseInput.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+<script type="text/javascript" src="http://cdn.nimiq.com/web-offline.js"></script>
+<script type="text/javascript" src="../src/lib/i18n.js"></script>
+<script type="text/javascript" src="../src/components/PassphraseInput.js"></script>
+<script type="text/javascript">
+    i18n.setLanguage('en');
+    translations.en.Confirm = 'Confirm';
+
+    const passphraseSetter = new PassphraseInput(true);
+    const passphraseGetter = new PassphraseInput();
```
Comment (Bettelstab) [inline demos/PassphraseInput.html]: add focus?

File: demos/PassphraseInput.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    const passphraseSetterContainer = document.getElementById('passphrase-setter-container');
+    const passphraseGetterContainer = document.getElementById('passphrase-getter-container');
+    passphraseGetterContainer.style.display = 'none';
+
+    passphraseSetterContainer.appendChild(passphraseSetter.getElement());
+    passphraseGetterContainer.appendChild(passphraseGetter.getElement());
+
+    let chosenPassphrase;
+    passphraseSetter.on(PassphraseInput.Events.PASSPHRASE_ENTERED, passPhrase => {
```
Comment (Bettelstab) [inline demos/PassphraseInput.html]: passPhrase => passphrase

File: src/components/PassphraseInput.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    cursor: pointer;
+}
+
+.passphrase-input button {
+    margin-top: 2rem;
+}
+
+.passphrase-input .strength-indicator-container {
+    width: 100%;
+    padding: 16px 0;
```
Comment (Bettelstab) [inline src/components/PassphraseInput.css]: px

File: src/components/PassphraseInput.css [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+.passphrase-input .strength-indicator-container .label {
+    text-align: center;
+}
+
+.passphrase-input .strength-indicator {
+    width: 100%;
+    height: 6px;
```
Comment (Bettelstab) [inline src/components/PassphraseInput.css]: px

**Comment by danimoh** [comment]
I addressed your change requests Max

**Pull request merged**